content/docs/latest/provisioning/ignition/boot-process.md

```diff
@@ -111,6 +111,22 @@ Here is an example config with an additional `links` entry that ensures that the
 }
 ```
 
+#### Debugging Ignition
+
+In the event of Ignition failure, there are most of the time three scenarios:
+* Instance console access with emergency shell
+In this case, you should be able to quickly identify the error (provider not implemented, unable to fetch configuration from the metadata service, unable to fetch a remote resource). From the emergency shell, it's possible to query the Ignition status with `journalctl -u ignition*`.
```

Comment:
```suggestion
In this case, you should be able to quickly identify the error (malformed config, provider not implemented, unable to fetch configuration from the metadata service, unable to fetch a remote resource). From the emergency shell, it's possible to query the Ignition status with `journalctl -u ignition*`.
```
I think it's good to mention the config error as this relates to the information that follows at the end.